enterprise/frontend/src/metabase-enterprise/data-studio/app/pages/LibrarySectionLayout/LibrarySectionLayout.tsx

```diff
@@ -98,47 +97,50 @@ export function LibrarySectionLayout() {
     error: snippetsError,
   } = useBuildSnippetTree();
 
-  const filteredTree = useTreeFilter({
-    data: [...tablesTree, ...metricsTree, ...snippetTree],
-    searchQuery,
-    searchProps: ["name"],
-  });
+  const combinedTree = useMemo(
+    () => [...tablesTree, ...metricsTree, ...snippetTree],
+    [tablesTree, metricsTree, snippetTree],
+  );
 
   const isLoading = loadingTables || loadingMetrics || loadingSnippets;
   useErrorHandling(tablesError || metricsError || snippetsError);
-  const filterReturnedEmpty = !!searchQuery && filteredTree.length === 0;
-  const showEmptyState = !isLoading && filterReturnedEmpty;
 
-  const libraryColumnDef = useMemo<ColumnDef<TreeItem, ReactNode>[]>(
+  const libraryHasContent = combinedTree.some(
+    (node) => node.children && node.children.length > 0,
+  );
+
+  const libraryColumnDef = useMemo<TreeTableColumnDef<TreeItem>[]>(
     () => [
       {
-        accessorKey: "name",
+        id: "name",
         header: t`Name`,
-        meta: { width: "auto" },
-        cell: ({ getValue, row }) => {
-          const data = row.original;
-          return (
-            <Group data-testid={`${data.model}-name`} gap="sm">
-              {data.icon && <Icon name={data.icon} c="brand" />}
-              {getValue()}
-            </Group>
-          );
-        },
+        enableSorting: true,
+        accessorKey: "name",
+        cell: ({ row }) => (
+          <EntityNameCell
+            data-testid={`${row.original.model}-name`}
+            icon={row.original.icon}
+            name={row.original.name}
+          />
+        ),
       },
       {
-        accessorKey: "updatedAt",
+        id: "updatedAt",
         header: t`Updated At`,
+        accessorKey: "updatedAt",
+        enableSorting: true,
+        sortingFn: "datetime",
+        width: 200,
```

Comment:
It's very easy to have the date time be too large for the column by using localization setitings. It appears to get cut off the cell
![alt text](image.png)

Reply:
<img width="904" height="830" alt="Image" src="https://github.com/user-attachments/assets/47efbaaa-3ee5-44f9-b74b-25dc7033dbdf" />

Reply:
Good catch, fixed it, thanks!